oops/oops.go

```diff
@@ -80,6 +111,9 @@ func writeSingleFrameTrace(b *strings.Builder, frames []Frame) {
 		b.WriteString(strconv.Itoa(frame.Line))
 		b.WriteRune('\n')
 	}
+	if framesSkipped {
+		b.WriteString("\tsubsequent stack frames truncated\n")
```

Comment:
What do you think about:
```
b.WriteRune('\t')
b.WriteString("subsequent stack frames truncated")
b.WriteRune('\n')
```
Not convinced, it seems less efficient, but also is easier to read and looks more like the other lines here. Is there something better than both?